src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] It would be better if you write your actions following  the [ducks pattern](https://github.com/erikras/ducks-modular-redux) so I encourage you to do that